docs/changelog.rst

```diff
@@ -3,9 +3,9 @@ Changelog
 
 .. scriv-insert-here
 
-.. _changelog-3.3.0:
+.. _changelog-3.3.1:
 
-globus-compute-sdk & globus-compute-endpoint v3.3.0
+globus-compute-sdk & globus-compute-endpoint v3.3.1
```

Comment:
FWIW, these weren't added in 3.3.1, but in 3.3.0.  Not a blocker, but being pedantic, I'd think this doesn't change.  Given that there's no apparent change between 3.3.0 and 3.3.1, then for general transparency, you might consider instead adding a changelog item that mentions why the new patch is necessary at all.  "Packaging mixup."

Reply:
Agreed; fixed